dynamic-layers/openembedded-layer/recipes-extended/images/tb-minimal-image.bb

```diff
@@ -13,6 +13,11 @@ IMAGE_INSTALL_append = " \
   grub-tb-common \
   dhcp-client \
   cbmem \
+  safeboot \
+  tpm2-totp \
```

Comment:
Is safeboot already RDPENDS on some of them, there is no need to provide them here as well.

Reply:
Fixed.